main.tf

```diff
@@ -43,7 +43,10 @@ resource "aws_instance" "this" {
   )
 
   provisioner "remote-exec" {
-    inline = ["echo 'sshd is running'"]
+    inline = [
+      "echo 'sshd is running'",
```

Comment:
Same as azure-rancher, this line isn't necessary anymore (was just to confirm the SSH connection)